src/app/features/pomodoro/dialog-pomodoro-break/dialog-pomodoro-break.component.ts

```diff
@@ -58,9 +59,14 @@ export class DialogPomodoroBreakComponent {
     this.isStopCurrentTime$.next(true);
     if (isSkipBreak) {
       this.pomodoroService.skipBreak();
+      this.pomodoroService.startBreak(false);
```

Comment:
This looks counter-intuitive. Could you maybe explain why a break is "started" when it is skipped?

Reply:
Yeah, you're right. I found it counter-intuitive myself. Now, that you confirmed my doubts I realised the issue was the [bad filtering predicate](https://github.com/jasyn/super-productivity/blob/da92c88e43e9347bc39f3ee4c52d5ed0139eccb4/src/app/features/pomodoro/pomodoro.service.ts#L92) I had on the `tick$` observable that didn't take into account whether it was a break or not. As a result, I was fixing the symptom by triggering the `startBreak(false)` to reset `isManualPauseBreak` to false. Also, this allows me to get rid of an unnecessary filter I had on the [notification pipe](https://github.com/jasyn/super-productivity/blob/da92c88e43e9347bc39f3ee4c52d5ed0139eccb4/src/app/features/pomodoro/store/pomodoro.effects.ts#L220-L223). Thanks!